src/contexts/NeynarContextProvider.tsx

```diff
@@ -62,20 +61,6 @@ export const NeynarContextProvider: React.FC<NeynarContextProviderProps> = ({
     );
   };
 
```

Comment:
Removal of the following code is a breaking change in the SDK.

It'll not add `.theme-light` or `.dark-theme` class to the root thus the theme properties associated with these themes (i.e. `--palette-background`, `--palette-border` etc) will not be present in the DOM.

`withThemeByClassName` will only work for storybook but we need these properties to be present in the DOM as well.

Is there a better way to add these classes to the root of the DOM?
```
useEffect(() => {
    const root = document.querySelector(":root");
    if (root) {
      if (theme === "light") {
        root.classList.add("theme-light");
        // root.classList.remove("theme-dark");
      }
      // else {
      //   root.classList.add("theme-dark");
      //   root.classList.remove("theme-light");
      // }
    }
  }, [theme]);
  ```